lang/src/org/partiql/lang/eval/builtins/BuiltinFunctions.kt

```diff
@@ -52,7 +52,7 @@ internal fun createBuiltinFunctions(valueFactory: ExprValueFactory) =
         SizeExprFunction(valueFactory),
         FromUnixTimeFunction(valueFactory),
         UnixTimestampFunction(valueFactory)
-    )
+    ) + MathFunctions.create(valueFactory)
```

Comment:
Could we also add some documentation similar to what we have in [BuiltInFunctions.md](https://github.com/partiql/partiql-lang-kotlin/blob/ecbaebc70b66d437d5ebed53e31b4c4e8315aeca/docs/user/BuiltInFunctions.md)? Feel free to create another markdown file for just these math functions.

Reply:
I will create a document for the math functions since this will grow soon.